packages/dashboard/src/components/task-alert.tsx

```diff
@@ -0,0 +1,218 @@
+import {
+  Box,
+  Button,
+  Grid,
+  LinearProgress,
+  LinearProgressProps,
+  Theme,
+  Typography,
+  Divider,
+  TextField,
+  useTheme,
+} from '@mui/material';
+import React from 'react';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import { makeStyles, createStyles } from '@mui/styles';
+import { RobotWithTask } from './task-alert-store';
+import { Status, TaskState } from 'api-client';
+import { RmfAppContext } from './rmf-app';
+import { Subscription } from 'rxjs';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    title: {
+      '&.MuiDialogTitle-root': {
+        padding: theme.spacing(1),
+        textAlign: 'center',
+        fontWeight: 'bold',
+      },
+    },
+    subtitle: {
+      '&.MuiTypography-root': {
+        fontWeight: 'bold',
+      },
+    },
+    textField: {
+      background: theme.palette.background.default,
+    },
+  }),
+);
+export interface AlertProps {
+  robots: RobotWithTask[];
+}
+
+function LinearProgressWithLabel(props: LinearProgressProps & { value: number }) {
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center' }}>
+      <Box sx={{ width: '100%', mr: 1 }}>
+        <LinearProgress variant="determinate" {...props} />
+      </Box>
+      <Box sx={{ minWidth: 35 }}>
+        <Typography variant="body2" color="text.secondary">{`${Math.round(
+          props.value * 100,
+        )}%`}</Typography>
+      </Box>
+    </Box>
+  );
+}
+
+interface AlertToDisplay extends RobotWithTask {
+  show: boolean;
+}
+
+export function AlertComponent({ robots }: AlertProps): JSX.Element {
+  const rmf = React.useContext(RmfAppContext);
+  const classes = useStyles();
+  const theme = useTheme();
+  const [index, setIndex] = React.useState(0);
+  const [alertToDisplay, setAlertToDisplay] = React.useState<AlertToDisplay[]>([]);
+
+  const setDialogColor = (taskStatus: Status | undefined) => {
+    if (taskStatus === undefined) return;
+
+    switch (taskStatus) {
+      case Status.Failed:
+        return theme.palette.error.dark;
+
+      case Status.Completed:
+        // TODO
+        // Add this color in the common-theme
```

Comment:
nit: todos can be written in a single line. Is there something blocking to have this color added to the theme?

Reply:
Ok, thanks for the suggestion. 
Oh, sorry. I forgot add it into common theme. 
I noticed that success.dark was not used anywhere, so I set #6dff6f instead.